src/components/TradingViewChart/datafeed.tsx

```diff
@@ -79,6 +79,7 @@ const TOKEN_PAIRS_ADDRESS_MAPPING: {
   '0x7ceb23fd6bc0add59e62ac25578270cff1b9f619': '0xce1923d2242bba540f1d56c8e23b1fbeae2596dc', // ETH_USD on polygon
   '0xff970a61a04b1ca14834a43f5de4533ebddb5cc8': '0x905dfcd5649217c42684f23958568e533c711aa3', // ETH_USD Arbitrum
   '0x7f5c764cbc14f9669b88837ca1490cca17c31607': '0x1a981daa7967c66c3356ad044979bc82e4a478b9', // ETH_USD Optimism
+  '0xe9e7cea3dedca5984780bafc599bd69add087d56': '0x58f876857a02d6762e0101bb5c46a8c1ed44dc16', // BNB_BUSD
```

Comment:
I wonder why `0x58f876857a02d6762e0101bb5c46a8c1ed44dc16` is [Pancake LP](https://bscscan.com/token/0x58f876857a02d6762e0101bb5c46a8c1ed44dc16) token? Is this correct?

Reply:
note that the right side addresses is Pair address in dextools api, it not relate to any smart contract address

Reply:
where could we find the documents for these addresses?

Reply:
It's better if we add it as a comment on top of this obj

Reply:
there is no document about dextools api, only can get that address by searching from their api or UI.